packages/wonder-blocks-toolbar/src/components/toolbar.tsx

```diff
@@ -73,74 +67,70 @@ type DefaultProps = {
-                    size === "small" && sharedStyles.small,
+                    sharedStyles.column,
+                    sharedStyles.leftColumn,
+                    title ? sharedStyles.withTitle : null,
                 ]}
             >
-                <View
-                    style={[
-                        sharedStyles.column,
-                        sharedStyles.leftColumn,
-                        title ? sharedStyles.withTitle : null,
-                    ]}
-                >
-                    {leftContent}
-                </View>
+                {leftContent}
+            </View>
 
-                {title && (
-                    <View
-                        style={[sharedStyles.column, sharedStyles.wideColumn]}
-                    >
-                        <View
-                            style={[sharedStyles.titles, sharedStyles.center]}
-                        >
-                            <TitleComponent id="wb-toolbar-title">
-                                {title}
-                            </TitleComponent>
-                            {subtitle && (
-                                <LabelSmall
-                                    style={
-                                        color === "light" &&
-                                        sharedStyles.subtitle
-                                    }
-                                >
-                                    {subtitle}
-                                </LabelSmall>
-                            )}
-                        </View>
+            {title && typeof title === "string" ? (
+                <View style={[sharedStyles.column, sharedStyles.wideColumn]}>
+                    <View style={[sharedStyles.titles, sharedStyles.center]}>
+                        <TitleComponent id="wb-toolbar-title">
+                            {title}
+                        </TitleComponent>
+                        {subtitle && (
+                            <LabelSmall
+                                style={
+                                    color === "light" && sharedStyles.subtitle
+                                }
+                            >
+                                {subtitle}
+                            </LabelSmall>
+                        )}
                     </View>
-                )}
-
-                <View
-                    style={[
-                        sharedStyles.column,
-                        sharedStyles.rightColumn,
-                        title ? sharedStyles.withTitle : null,
-                    ]}
-                >
-                    {rightContent}
                 </View>
+            ) : (
+                // We don't use wideColumn here to allow more flexibility with
+                // the custom node.
+                <View style={[sharedStyles.column]}>
+                    <View style={[sharedStyles.titles]}>{title}</View>
+                </View>
+            )}
```

Comment:
**note:** This is the actual change that allows for more flexibility to the `title` prop.